docs/src/tutorials/getting_started/performance_tips.jl

```diff
@@ -159,3 +159,40 @@ model = Model()
 # If we use the [`@expression`](@ref) macro, we get many fewer allocations:
 
 @allocated @expression(model, x[1] + x[2] + x[3])
+
+# ## Disable string names
+
+# By default, JuMP creates `String` names for variables and constraints and
+# passes these to the solver. The benefit of passing names is that it improves
+# the readability of log messages from the solver (e.g., "variable x has invalid
+# bounds" instead of "variable v1203 has invalid bounds"), but for larger models
+# the overhead of passing names can be non-trivial.
+
+# Disable the creation of `String` names by setting `set_string_name = false` in
+# the [`@variable`](@ref) and [`@constraint`](@ref) macros, or by calling
+# [`set_string_names`](@ref) to disable all names for a particular model:
+
+model = Model()
+set_string_names(model, false)
+@variable(model, x)
+
+#-
+
+@constraint(model, c, 2x <= 1)
+
+# Note that this doesn't change how symbolic names and bindings are stored:
```

Comment:
Perhaps also show that looking up the variable by string name will now give an error